spec/index.md

```diff
@@ -606,7 +606,7 @@ The HTTP response to this request can have the following responses:
 
 -   5xx HTTP status when the server has encountered an error or is otherwise incapable of performing the request. The request might succeed if retried at a later time.
 
-This request type can *also* be used to call a query method. A user may choose to go this way, instead of via the faster and cheaper [Request: Query call](#http-query) below, if they want to get a *certified* response. Note that the canister state will not be changed by sending a call request type for a query method.
+This request type can *also* be used to call a query method (but not a composite query method). A user may choose to go this way, instead of via the faster and cheaper [Request: Query call](#http-query) below, if they want to get a *certified* response. Note that the canister state will not be changed by sending a call request type for a query method.
```

Comment:
Won't the balance change, even if the rest of the state does not?

Reply:
I think canister state refers strictly to the canister memory and not system side state pertaining to the canister.

Reply:
I've added an explicit note:
```
Note that the canister state (except for cycle balance change due to message execution) will not be changed ...
```